src/main/resources/manifest.yml

```diff
@@ -1,3 +1,3 @@
-manifestVersion: 1.0.0-alpha
-productName: @serviceName@
-productVersion: @serviceVersion@
+manifest-version: 1.0.0
+product-name: @serviceName@
```

Comment:
Re release/merge: happy to postpone the merge, or just not make a release before other products support this